content/vendor/guides/existing-cluster/_index.md

```diff
@@ -52,6 +102,147 @@ Enter a new password to be used for the Admin Console: ••••••••
   • Go to http://localhost:8800 to access the Admin Console
 ```
 
-Access KOTSadm on `http://localhost:8800` and enter the password. After that you will see the License page similar to the Quickstart guide.
+### Install License
+
+At this point, kotsadm and Kubernetes are running, but the application isn't yet. This is also what your customer would be experiencing when installing your application. To complete the installation, visit the URL `http://localhost:8800` where you'll be required to enter the password set earlier.
+
+![Console TLS](/images/guides/kots/admin-console-login.png)
+
+Now the installation needs a license file to continue. Until this point, this cluster is just running Kubernetes and the kotsadm containers. Once we put a license file on it the application will install. Click the Upload button and select your `.yaml` file to continue.
 
 ![Upload License](/images/guides/kots/upload-license.png)
+
+The settings page is here with default configuration items. These can be specified in the `config.yaml` file.
+
+![Settings Page](/images/guides/kots/configuration.png)
+
+Preflight checks are designed to ensure this server has the minimum system and software requirements to run the application. By default we include some preflight checks that are expected to fail so that we can see what failing checks might look like for a customer. If you click continue it will warn you but you can still continue.
+
+![Preflight Checks](/images/guides/kots/preflight.png)
+
+Click the Application link on the top to see the application running. If you are still connected to this server over ssh, `kubectl get pods` will now show the example nginx service we just deployed.
+
+![Dashboard](/images/guides/kots/dashboard.png)
+
+On the nav bar, there's a link to the application page. Clicking that will show you the Kubernetes services that we just deployed.
+
+![Cluster](/images/guides/kots/application.png)
+
+### View the application
+
+To view the running Nginx Application, you can head to `http://localhost:8888/` and you should see a basic (perhaps familiar) nginx server running:
```

Comment:
we removed the auto-port forward from the default YAML, right? Should we have the user manually port-forward in this case, and then link to some docs on "doing this automatically" (about putting a button on the dashboard)?